book/01-introduction/sections/what-is-git.asc

```diff
@@ -83,7 +83,7 @@ Git has three main states that your files can reside in: _modified_, _staged_, a
 * Staged means that you have marked a modified file in its current version to go into your next commit snapshot.
 * Committed means that the data is safely stored in your local database.
 
-This leads us to the three main sections of a Git project: the working tree, the staging area, and the Git directory.
+This leads us to the three main sections of a Git project: the working directory, the staging area, and the Git directory.
 
 .Working tree, staging area, and Git directory
 image::images/areas.png["Working tree, staging area, and Git directory."]
```

Comment:
Unfortunately this change disagrees with the caption of the diagram right below it. The diagram itself says "Working Directory", so let's standardize on that. You'll want to update all of these locations.

![CleanShot 2022-10-05 at 11 00 38](https://user-images.githubusercontent.com/39902/194129858-a044b085-d4ed-4ad6-9298-91eb73235626.jpg)